File: 351001/shevchenkoAD/DistComp/DistComp/Domain/Entities/Author.cs
using DistComp.Domain.Abstractions;

namespace DistComp.Domain.Entities;

public class Author : BaseEntity {
    public string Login { get; set; } = "";
    public string Password { get; set; } = "";
    public string Firstname { get; set; } = "";
    public string Lastname { get; set; } = "";
}

File: 351004/Romanovskiy/task310/src/main/java/com/example/task310/domain/entity/Author.java
package com.example.task310.domain.entity;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import lombok.experimental.SuperBuilder;

@Data
@EqualsAndHashCode(callSuper = true)
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class Author extends BaseEntity {
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}